### DISCUSSION TITLE
Only admins can break things in the wilderness, explosions regenerate are regenerating.

### DISCUSSION THREAD

**Discussion by Dansupercutter** [TownyAdvanced/Towny#4721] [Q&A] [answered]
Only admins are able to break stuff in the wilderness, i want to know how to fix that and i want to know how to disable the regeneration of blocks when they are destroyed in the wilderness. I don't know if there is a file of these things im spose to look for if that is the case please do  tell me where it is

**Comment by LlmDl** [comment, marked as answer]
Hello, 

You've stumbled onto one of the [FAQs](https://github.com/TownyAdvanced/Towny/wiki/Frequently-Asked-Questions)

To reference it: 

>Q: Why can users not break or build in the wilderness outside of their towns?
>
>A: Read the [opening paragraphs of the nodes page.](https://github.com/TownyAdvanced/Towny/wiki/Towny-Permission-Nodes)

and 

`/tw toggle explblockregen`
